configs/testnet.json

```diff
@@ -14,5 +14,11 @@
     ],
     "blacklisted_pools": [],
     "blacklisted_tokens": [],
-    "ibc_tokens_total_supply": []
-}
+    "ibc_tokens_total_supply": [],
+    "transfer_options": {},
+    "network_fees": {
+      "swth": 0,
+      "usdc": 1,
+      "eth": 2
+    }
+  }
```

Comment:
Heyyo, for this PR:
1. You may need to add struct validation into `config.schema.json` like the rest of the fields.
  Can check out how it's done [here](https://json-schema.org/learn/)

2. Do update the `README.md` document with the fields that you added.

4. Do update JSON Data Structure section under `.github/markets/pr_template.md` with the definitions of the 2 new fields.

5. Under `scripts/check_configs.ts`, think it would be good to add code to validate the 2 new fields:
  - `transfer_options`
    - [ ] Whether or not the blockchain names (i.e. Cosmos Hub/Zilliqa) are valid bridges listed under `BridgeAll` RPC call
  - `network_fees`
    - [ ] Whether or not the denoms listed (i.e. swth/usdc) are valid denoms listed under `MinGasPriceAll` RPC call